build/Helix/CopyVisualTreeMasters.ps1

```diff
@@ -0,0 +1,6 @@
+Param(
+    [string]$Destination
```

Comment:
Can you add a help blurb? [https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-6](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-6)